1.1/spec/20-Annex-D.adoc

```diff
@@ -0,0 +1,260 @@
+
+
+== E.7 schema.org
+
+schema.org Source: https://schema.org
+
+|===
+| From Element | Mapping relation | To Element | Notes
+
+| `geo:Geometry` | `rdfs:subClassOf` | `sdo:GeoShape` | A GeoShape can various literal geometry representation
+| `sdo:GeospatialGeometry` | `owl:equivalentClass` | `geo:SpatialObject` | Since GeospatialGeometry is the domain of SimpleFeature-like properties and a superclass of GeoShape
+| `sdo:GeoCoordinates` | `rdfs:subClassOf` | `geo:Geometry` | GoCoordinates uses direct lat, long, elevation etc properties to indicate position, not a while geometry serialization but it is nevertheless a form of a Geometry
+| `sdo:geo` | `rdfs:subPropertyOf` | `geo:hasGeometry` |
+| `sdo:geoCoveredBy` | `owl:equivalentProperty` | `geo:ehCoveredBy` | 
+| `sdo:geoCovers` | `owl:equivalentProperty` | `geo:ehCovers` | 
+| `sdo:geoCrosses` | `owl:equivalentProperty` | `geo:sfCrosses` | 
+| `sdo:geoDisjoint` | `owl:equivalentProperty` | `geo:sfDisjoint` | 
+| `sdo:geoEquals` | `owl:equivalentProperty` | `geo:sfEquals` | 
+| `sdo:geoIntersects` | `owl:equivalentProperty` | `geo:sfIntersects` | 
+| `sdo:geoOverlaps` | `owl:equivalentProperty` | `geo:sfOverlaps` | 
+| `sdo:geoTouches` | `owl:equivalentProperty` | `geo:sfTouches` | 
+| `sdo:geoWithin` | `owl:equivalentProperty` | `geo:sfWithin` | 
+| `sdo:Landform` | `rdfs:subClassOf` | `geo:Feature` |
+|===
+
+
+== E.8 Semantic Sensor Network Ontology (SSN)
+
+SSN Source: https://www.w3.org/TR/vocab-ssn/
+
+SSN and GeoSPARQL do not cover overlapping concerns directly and therefore there are no direct class or property correspondences between them, however SSN provides advice on the use of GeoSPARQL for location, 
+see Section 7.1 (https://www.w3.org/TR/vocab-ssn/#x7-1-location):
+
+> GeoSPARQL ... provides a flexible and relatively complete platform for geospatial objects, that fosters interoperability between geo-datasets. To do so, these entities can be 
+declared as instances of geo:Feature and geometries can be assigned to them via the geo:hasGeometry property. In case of classes, e.g., specific features of interests such as 
+rivers, these can be defined as subclasses of geo:Feature.
+
+
+== E.9 DCMI Metadata Terms (DCTERMS)
+
+DCTERMS Source: https://www.dublincore.org/specifications/dublin-core/dcmi-terms/
+
+|===
+| From Element | Mapping relation | To Element | Notes
+
+| `geo:Feature` | `rdfs:SubClassOf` | `dcterms:Location` | A Location is a "A spatial region or named place."
+| `dcterms:coverage` - | - | coverage is extremely generic - "The spatial or temporal topic of the resource, spatial applicability of the resource, or jurisdiction under which the resource is relevant." - but DCTERMS indicates its range includes a `dcterms:Location`, so it is a property for indicating a `geo:Feature`, not a `geo:Geometry` and for which GeoSPARQL has no equivalent
```

Comment:
More often `dcterms:coverage` is used to indicate extent, usually a bounding-box. There is probably not a formal mapping that can be asserted, but perhaps draw attention to bounding-box and Envelope here, since we've added it.

Reply:
Added